extensions/disruptor/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -24,9 +24,8 @@
 name: "Camel Disruptor"
 description: "Provides asynchronous SEDA behavior using LMAX Disruptor"
 metadata:
-  unlisted: true
   guide: "https://camel.apache.org/camel-quarkus/latest/reference/extensions/disruptor.html"
   categories:
   - "integration"
   status:
-  - "preview"
+  - "stable"
```

Comment:
Is it really stable when we just landed native support?

Reply:
@ppalaga what do you think ? I don't remember the rationale

Reply:
We currently consider all extensions stable that are native and have a proper test.